### ISSUE TITLE
Plan v5.1 build 474  Could not pass event PlayerViolationCommandEvent

### ISSUE THREAD

**Issue by Haczyn** [plan-player-analytics/Plan#1374]
```
[02:23:04 ERROR]: Could not pass event PlayerViolationCommandEvent to Plan v5.1 build 474
java.lang.IllegalArgumentException: null
at java.util.Optional.orElseThrow(Optional.java:290) ~[?:1.8.0_242]
at me.konsolas.aac.e.getViolationLevel(e.java) ~[?:?]
at com.djrapitops.extension.AACHackKickListener.onJoin(AACHackKickListener.java:66) ~[?:?]
at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor518.execute(Unknown Source) ~[?:?]
at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.15.2.jar:git-Paper-143]
at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.15.2.jar:git-Paper-143]
at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.15.2.jar:git-Paper-143]
at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:559) ~[patched_1.15.2.jar:git-Paper-143]
at me.konsolas.aac.cl.a(cl.java) ~[?:?]
at me.konsolas.aac.Y.run(Y.java) ~[?:?]
at org.bukkit.craftbukkit.v1_15_R1.scheduler.CraftTask.run(CraftTask.java:84) ~[patched_1.15.2.jar:git-Paper-143]
at org.bukkit.craftbukkit.v1_15_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:452) ~[patched_1.15.2.jar:git-Paper-143]
at net.minecraft.server.v1_15_R1.MinecraftServer.b(MinecraftServer.java:1185) ~[patched_1.15.2.jar:git-Paper-143]
at net.minecraft.server.v1_15_R1.DedicatedServer.b(DedicatedServer.java:430) ~[patched_1.15.2.jar:git-Paper-143]
at net.minecraft.server.v1_15_R1.MinecraftServer.a(MinecraftServer.java:1112) ~[patched_1.15.2.jar:git-Paper-143]
at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:934) ~[patched_1.15.2.jar:git-Paper-143]
at java.lang.Thread.run(Thread.java:748) [?:1.8.0_242]
```

When i update version from Plan-5.0-dev-build-458 to Plan v5.1 build 474. I clear all conf, and still same error.

**Comment by Rsl1122** [comment]
Try updating your AAC to a newer version or disable the AAC integration in the Plan config.

**Comment by Rsl1122** [comment]
Fixed in AAC 4.3.9
